### PR TITLE
multiple states for nws_convert

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
fixes #55

File: viz.yaml [hunk 1]
```diff
@@ -215,7 +215,7 @@ fetch:
     id: nws-conversion
     location: cache/nws-conversion.rds
     reader: rds
-    states: ["FL"] # not currently coded to support more...
+    states: ["FL", "GA"]
```
Comment (dblodgett-usgs) [inline viz.yaml:218]: We should figure out how to have one list of states for the whole viz. We could derive this list from the polygon that's passed in for the storm area intersected with a list of states?

**Comment by dblodgett-usgs** [comment]
👍  I've got an idea to clean up the multi-state selection across the viz that would be able to generate the list of states for this. Let's 🚢  this and clean that up later though.

File: viz.yaml (hunk 1 above)
Comment (wdwatkins) [reply viz.yaml:218]: Yes, that would be good.

**Pull request merged**